ra-optimisation/linear-range-action-rao/src/test/java/com/farao_community/farao/linear_range_action_rao/LinearRangeActionRaoTest.java

```diff
@@ -225,21 +223,17 @@ private static SimpleCrac create() {
 
         ComplexRangeAction rangeAction2 = new ComplexRangeAction("idRangeAction2", "myRangeAction2", "operator1", usageRules, ranges, new ArrayList<>(Arrays.asList(pstRange1)));
 
-        List<Cnec> cnecs = new ArrayList<>();
-        cnecs.add(cnec1);
+        Crac crac = new SimpleCrac("idCrac", "name");
 
-        SimpleCrac crac = new SimpleCrac("idCrac", "name", new ArrayList<>(), new ArrayList<>(), new ArrayList<>());
-
-        crac.setCnecs(cnecs);
+        crac.addCnec(cnec1);
         crac.addCnec(cnec2);
-        crac.setNetworkActions(new ArrayList<>(Arrays.asList(networkAction1)));
-        crac.addNetworkRemedialAction(networkAction2);
-        crac.setRangeActions(new ArrayList<>(Arrays.asList(rangeAction1)));
-        crac.addRangeRemedialAction(rangeAction2);
-
-        String branchId = "BBE2AA1  BBE3AA1  1";
-        ComplexContingency contingency1 = new ComplexContingency("idContingency", "My contingency",
-                Arrays.asList(new NetworkElement("BBE2AA1  BBE3AA1  1", "BBE2AA1  BBE3AA1  1")));
+        crac.addNetworkAction(networkAction1);
+        crac.addNetworkAction(networkAction2);
+        crac.addRangeAction(rangeAction1);
+        crac.addRangeAction(rangeAction2);
+
+        ComplexContingency contingency1 = new ComplexContingency("idContingencyTmp", "My contingency Tmp",
+                new HashSet<>(Arrays.asList(new NetworkElement("BBE2AA1  BBE3AA1  1", "BBE2AA1  BBE3AA1  1"))));
```

Comment:
With crac refactoring you can only put an id (when id and name are equals)

Reply:
ok